src/devices/bus/coco/coco_t4426.cpp

```diff
@@ -0,0 +1,195 @@
+// license:BSD-3-Clause
+// copyright-holders:Joakim Larsson Edstrom
+/***************************************************************************
+ *
+ *  coco_t4426.c
+ *
+ *  Terco T4426 CNC Programming Station multi cart
+ *
+ *  The code here is heavily inspired by coco_pak and coco_232
+ *
+ *  +-------------------------------------------------------------------------------+
+ *  ||__|+-----+    oo   75                               |O ||||||||||||||| O|     |
+ *  |XTL||MC   |    oo  110                                                         |
+ *  |1.8||14411|    oo  ..        +--+ +--+                                         |
+ *  |432||BRG  |    ==  600       |74| |74| +------+                                |
+ *  |MHz||     |    oo  ..        |LS| |LS| |MC1488|                                |
+ *  +---+|     |    oo 7200       |139 |00| +------+                                |
+ *  |    +-----+    oo 9600       +--+ +--+                    +--+                 |
+ *  |                                      +-------------+     |MC|                 |
+ *  |    +-----+   +-----+    +-----+      | EF68B50P    |     |14|                 |
+ *  |    | 2764|   | 2764|    |     |      | ACIA        |     |89|                 |
+ *  |    |     |   |     |    |     |      +-------------+     +--+                 |
+ *  |    |CA   |   |CA   |    |PMOS |                                               |
+ *  |    | 4426|   | 4426|    | 4426|   +-------------------+                       |
+ *  |    |  -6 |   |  -7 |    |     |   |  EP68B21P         |                       |
+ *  |    |     |   |     |    |     |   |  PIA              |                       |
+ *  |    |     |   |     |    +-----+   +-------------------+                       |
+ *  |    +-----+   +-----+                                                          |
+ *  |    +-----+   +-----+    +-----+   +-----+    +-----+   +-----+                |
+ *  |    | 2764|   | 2764|    | 2764|   | 2764|    | 2764|   | 2764|                |
+ *  |    |     |   |     |    |     |   |     |    |     |   |     |                |
+ *  |    |CA   |   |CA   |    |PD   |   |PD   |    |ED   |   |ED   |                |
+ *  |    | 4426|   | 4426|    | 4426|   | 4426|    | 4426|   | 4426|                |
+ *  |    |  -5 |   |  -4 |    |  -3 |   |  -2 |    |  -1 |   |  -0 |                |
+ *  |    |     |   |     |    |     |   |     |    |     |   |     |       OO       |
+ *  |    |     |   |     |    |     |   |     |    |     |   |     |                |
+ *  |    +-----+   +-----+    +-----+   +-----+    +-----+   +-----+                |
+ *  +-------------------------------------------------------------------------------+
+ *
+ ***************************************************************************/
+
+#define VERBOSE 0
+
+#define LOGPRINT(x) do { if (VERBOSE) logerror x; } while (0)
+#define LOG(x)      {} LOGPRINT(x)
```

Comment:
Another issue with these is they behave as a single statement when disabled but as two statements when enabled.  This will make trouble with things like:
```
if (foo)
    LOG(something);
```
Intuitively one would expect it to only log something if `foo` evaluates to true, but it will actually always log.  Also, consider this:
````
if (foo)
    LOG(something);
else
    action();
```
This won't compile at all if the `LOG` macro is enabled.

This kind of behaviour is unintuitive.  Software should not be surprising, it should do what the developer and/or user expect.  I don't want to see these kinds of patterns spread further in the codebase.